libs/s25main/figures/noFigure.cpp

```diff
@@ -397,7 +393,9 @@ void noFigure::HandleEvent(const unsigned id)
         switch(fs)
         {
             case FS_GOHOME:
-            case FS_GOTOGOAL: { WalkToGoal();
+            case FS_GOTOGOAL:
+            {
```

Comment:
IndentCaseLabels=false and probably  IndentCaseLabels=false ?

Reply:
C&P bug in your message? ;)

Reply:
second one should be `IndentCaseBlocks=false` ;-)

Reply:
That doesn't exist in 9: https://releases.llvm.org/9.0.0/tools/clang/docs/ClangFormatStyleOptions.html

Anyway I'd keep it like it is as it's the most consistent and IMO most readable.